package.json

```diff
@@ -51,16 +51,20 @@
     "read-dir-files": "^0.1.1",
     "standard": "^16.0.3"
   },
-  "main": "./lib/index.js",
+  "exports": {
+    ".": "./lib/index.js",
+    "./esm": "./lib/esm.mjs"
+  },
```

Comment:
Have you considered using export conditions so it automatically exports the ESM version when `import` is used instead of `require`?

```
"exports": {
	".": {
		"require": "./lib/index.js",
		"import": "./lib/esm.mjs"
	},
	"./esm": "./lib/esm.mjs"
}
```

I added the `./esm` in case, but this way, you may not even need the `./esm` endpoint:

```
"exports": {
	"require": "./lib/index.js",
	"import": "./lib/esm.mjs"
}
```

Reply:
I don't want to do that, because the ESM version doesn't export normal `fs` methods like the CJS version does. Since it's a different feature set, it's best to be explicit. Otherwise, I'd definitely do this.

Reply:
Why the regression? Sounds it can be kept with:
```ts
export * from 'fs';
```

Reply:
See https://github.com/jprichardson/node-fs-extra/issues/746#issuecomment-885076808